src/javascripts/components/Chat.js

```diff
@@ -2,127 +2,78 @@ import React from 'react'
 import ChatBoxSettings from './ChatBoxSettings'
 import ChatBox from './ChatBox'
 import ChatInput from './ChatInput'
-import { resMsgs } from '../helper'
+import resMsgs from '../helper'
 
 class Chat extends React.Component {
   constructor() {
-      super()
-      this.state = {
-        messages: [
-          {
-            user: "blue",
-            message: "blah blah blah"
-          }, 
-          {
-            user: "yellow",
-            message: "Lorem ipsum dipsum"
-          },
-          {
-            user: "yellow",
-            message: "Hallo lorem"
-          }
-        ]
-      }
-  }  
+    super()
+    this.state = {
+      messages: [
+        {
+          user: 'blue',
+          message: 'blah blah blah'
+        },
+        {
+          user: 'yellow',
+          message: 'Lorem ipsum dipsum'
+        },
+        {
+          user: 'yellow',
+          message: 'Hallo lorem'
+        }
+      ]
+    }
+  }
 
   componentWillMount() {
-    const localStorageRef = localStorage.getItem(`messages`)  
-    if(localStorageRef) {
+    const localStorageRef = localStorage
+      ? localStorage.getItem('messages')
+      : null
```

Comment:
I'm not a fan of ternaries like this on multiple lines, sorry :/

If a ternary gets loooog, then you may as well just make it an if/else for clarity.  Sure, it's more lines of code, but it's easier to read.